felix/bpf/xdp/attach.go

```diff
@@ -61,7 +68,7 @@ func (ap AttachPoint) FileName() string {
 }
 
 func (ap AttachPoint) SectionName() string {
-	return "calico_entrypoint_xdp"
+	return "xdp/calico_entrypoint"
```

Comment:
Can't this be just ```calico_entrypoint_xdp```? I think we don't use ```classifier``` when loading ```tc``` programs

Reply:
if I remove `xdp` from the beginning the libbpf cannot detect its type and ut tests fails. I tried few combination, but all fails to load the program.